apps/filebrowser/src/filebrowser/api.py

```diff
@@ -250,49 +250,75 @@ def listdir_paged(request):
   A paginated version of listdir.
 
   Query parameters:
-    pagenum           - The page number to show. Defaults to 1.
-    pagesize          - How many to show on a page. Defaults to 30.
-    sortby=?          - Specify attribute to sort by. Accepts: (type, name, atime, mtime, size, user, group). Defaults to name.
-    descending        - Specify a descending sort order. Default to false.
-    filter=?          - Specify a substring filter to search for in the filename field.
+    pagenum (int): The page number to show. Defaults to 1.
+    pagesize (int): How many items to show on a page. Defaults to 30.
+    sortby (str): The attribute to sort by. Valid options: 'type', 'name', 'atime', 'mtime', 'user', 'group', 'size'.
```

Comment:
Why default to type rather than name? Isn't it more common to sort by name?

Reply:
I checked Google drive as reference and there by default it was sorting by type (file or folders).

Reply:
Verified again and turns out there was extra dropdown value in Google drive which was explicitly set to sort based on type or show folders first, removing that it sorts by name by default. 

So I'm switching back to name as default in our code as well.